sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -411,7 +408,11 @@ pub(crate) fn sdr_not_expired(output: &Output, current_time: u32) -> Option<&Sto
                 .map_or(false, |expiration| current_time >= expiration.timestamp());
 
             // We only have to send the storage deposit return back if the output is not expired
-            if !expired { Some(sdr) } else { None }
+            if !expired {
+                Some(sdr)
+            } else {
+                None
+            }
```

Comment:
```suggestion
            !expected.then_some(sdr)
```

Reply:
You probably meant `(!expired).then_some(sdr)` but ok